lib/stripe/stripe_configuration.rb

```diff
@@ -27,7 +27,6 @@ module Stripe
   class StripeConfiguration
     attr_accessor :api_key
     attr_accessor :api_version
-    attr_accessor :client_id
```

Comment:
This one snuck in on accident since `client_id` is still set on the Stripe object:

https://github.com/stripe/stripe-ruby/blob/0620436d63a40af8bdc93057c1cdb4e58bf1a30a/lib/stripe.rb#L87

If we want this to be figured per client, I'll also need to update the OAuth code to account for it.